OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/NewCourseOutlineAdapter.java

```diff
@@ -771,4 +814,22 @@ public int getPositionByItemId(String itemId) {
         }
         return -1;
     }
+
+    // TODO: Remove this code, its just for performance testing purpose
+    long startTime = System.currentTimeMillis();
+    long endTime = System.currentTimeMillis();
+    int timesCalledInASec = 0;
+
+    @Override
+    public void notifyDataSetChanged() {
```

Comment:
Add a todo over this function as well to remove this after testing

Reply:
There is already a TODO above this line:
`long startTime = System.currentTimeMillis();`

To make it more clear I've removed the extra space below `int timesCalledInASec = 0;`.